examples/using-mobx/wrap-with-provider.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Provider} from 'mobx-react'
+import CounterStore from "./src/models/CounterModel"; 
+
+// eslint-disable-next-line react/display-name,react/prop-types
+export default({element})=><Provider store={CounterStore}>{element}</Provider>
```

Comment:
only "gotcha" here would be that "dynamic" path that would be later required on client would create "require context" that include a lot of files in produce huge bundle size

this can be handled same way I did for layout plugin: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-layout/src/gatsby-node.js so during compilation with end up with string literal instead of path variable used to require store file - but this approach force limit of only one mobx plugin instance in configuration - which probably makes sense but not sure :)